packages/gatsby-source-strapi/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "gatsby-source-strapi",
+  "version": "3.0.0",
+  "description": "Gatsby source plugin for building websites using Strapi as a data source",
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source-plugin",
+    "strapi"
+  ],
+  "directories": {
+    "dist": "./dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsby-uc/plugins.git",
+    "directory": "packages/gatsby-source-strapi"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsby-uc/plugins/issues"
+  },
+  "homepage": "https://github.com/gatsby-uc/plugins/packages/gatsby-source-packagist#readme",
+  "author": {
+    "email": "hi@strapi.io",
+    "name": "Strapi Solutions",
+    "url": "http://strapi.io"
+  },
+  "contributors": [
+    "Laurens Kling <laurens@goedideemeida.nl>"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "axios": "0.24.0",
+    "commonmark": "^0.30.0",
+    "gatsby-source-filesystem": "4.5.1",
+    "lodash": "^4.17.21",
+    "qs": "^6.10.3"
+  },
+  "scripts": {
+    "build": "babel src --out-dir dist"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.14.5",
+    "@babel/core": "7.14.6"
+  },
+  "peerDependencies": {
+    "gatsby": "^4.0.0 || ^5.0.0"
```

Comment:
The image plugin and filesystem packages that were removed may need adding to deps if they're being directly imported anywhere

Reply:
No only `import { createRemoteFileNode } from "gatsby-source-filesystem";` is used in the codebase itself, and gatsby-source-filesystem is in the deps. I\ll update it to 5 (i assume it doesn't have a breaking change in `createRemoteFileNode`) to support Gatsby 5 better. The image and sharp peer deps are a bit weird, because you should be able to use this source without sourcing any images.